### ISSUE TITLE
Bug: Can't see the entire reaon of failure

### ISSUE THREAD

**Issue by alexisfrjp** [ArchiveBox/ArchiveBox#907]
#### Describe the bug

On the log webpage, we can't see the reason.

#### Screenshots or log output

![image](https://user-images.githubusercontent.com/6713988/146736395-645a6e76-7dae-4904-8264-b576fe375342.png)

No possibility to scroll down.

#### ArchiveBox version

![image](https://user-images.githubusercontent.com/6713988/146736513-1c92ad3b-477f-47a0-ac62-c5802e4c3fd1.png)

**Comment by AndTheDaysGoBy** [comment]
Might've been a purposeful choice? Fiddling in browser, overflow was set to hidden. Checking `admin.css` file, we have
https://github.com/ArchiveBox/ArchiveBox/blob/a9986f1f05bfcda8cbb6b7c915854560f98d3e3e/archivebox/templates/static/admin.css#L304
where it's specifically set to `hidden` instead of `scroll`.

**Comment by pirate** [comment]
Yeah, my reasoning was that having many dozens of nested scrollbars is kind of annoying, and if you need the full output you can ctrl+A , ctrl+c, ctrl+v somewhere else.

**Comment by alexisfrjp** [comment]
If it's annoying, better not to show them at all. What's annoying is to have texts that can't be seen entirely...
Ctrl+A doesn't work, I guess it must depend on browsers.

**Comment by pirate** [comment]
Try triple click
